.github/workflows/integration_test.yaml

```diff
@@ -19,66 +18,24 @@ jobs:
             value: "--num-units=3"
           - name: num_units=1, db_from_relation
             value: "--num-units=1"
-        exclude:
-          - module: test_error
-            args:
-              name: num_units=3, db_from_config
-              value: "--test-db-from-config --num-units=3"
-          - module: test_error
-            args:
-              name: num_units=1, db_from_config
-              value: "--test-db-from-config --num-units=1"
-          - module: test_upgrade
-            args:
-              name: num_units=3, db_from_relation
-              value: "--num-units=3"
-          - module: test_upgrade
-            args:
-              name: num_units=1, db_from_relation
-              value: "--num-units=1"
-
-    name: Integration Tests (${{ matrix.module }}, ${{ matrix.args.name }})
-
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    with:
+      setup-devstack-swift: true
+      pre-run-script: |
+        -c "sudo microk8s config > kube-config && \
```

Comment:
Perhaps we can move the first command to https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml

Reply:
Ah the charmed kubernetes action already does seem to output the kubeconfig file to ~/.kube/config(according to the source code)! Let me create a pull request documenting the behavior, while this can be updated to use the kube config file from $HOME/.kube/config.
Thank you for the heads up!